src/command_modules/azure-cli-acr/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+2.3.1
++++++
+* Marking this package as deprecated. Please use azure-cli instead.
```

Comment:
I would recommend wording like:
* Final release to mark this package as deprecated. Future updates can be found in the `azure-cli` package instead.

Reply:
Fixed in e92d6032